components/gripper/trossen/trossen.go

```diff
@@ -3,58 +3,54 @@ package trossen
 
 import (
 	"context"
-	"sync"
-	"time"
 
 	"github.com/edaniels/golog"
-	"github.com/jacobsa/go-serial/serial"
-	"go.uber.org/multierr"
-	"go.viam.com/dynamixel/network"
-	"go.viam.com/dynamixel/servo"
-	"go.viam.com/dynamixel/servo/s_model"
+	"github.com/pkg/errors"
 	"go.viam.com/utils"
 
+	"go.viam.com/rdk/components/arm"
+	trossenarm "go.viam.com/rdk/components/arm/trossen"
 	"go.viam.com/rdk/components/generic"
 	"go.viam.com/rdk/components/gripper"
 	"go.viam.com/rdk/config"
-	"go.viam.com/rdk/operation"
 	"go.viam.com/rdk/referenceframe"
 	"go.viam.com/rdk/registry"
 	"go.viam.com/rdk/resource"
 	rdkutils "go.viam.com/rdk/utils"
 )
 
+// This is an implementation of the Gripper API for Trossen arms. The gripper
+// requires a single Trossen arm component in its dependencies upon configuration,
+// as all of the gripper functions defer to DoCommand on the arm implementation. See
+// components/arm/trossen/trossen.go for more about the arm implementations
+// and supported arms
+
 var (
 	modelNameWX250s = resource.NewDefaultModel("trossen-wx250s")
 	modelNameVX300s = resource.NewDefaultModel("trossen-vx300s")
```

Comment:
[opt] If we're doing breaking changes, why two model names? Why not just "trossen"? Would reduce code even further (no duplicate registrations, etc.)

Reply:
Pinging @Fahmina concerning this to double-check, but we apparently want to move towards multiple specific model names pointing to a singular implementation because it's more transparent concerning what models of arms we actually support. In this case it's also important because I believe these are the only two trossen arms for which this code will work

Reply:
just as an example for this PR, the kinematics are different, and resolved into frames differently, and the only user-facing way we have of ingesting that information is by the two models names.

trossen themselves don't have a sense of different arm models registering differently. They do this for servos, and the kinematics are developed on ROS/matlab/LABVIEW implementations. So I don't think we can generalise these two models down to one anyways.

Reply:
We can add it as an attribute later if needed, but too many changes to the code structure currently. @gvaradarajan can you please make a product ticket so @Fahmina can think about this?

Reply:
eventually... there will be frames, frames everywhere (some grippers already implement them). And yes, sorry, just realised it was optional. Agree with you - best to leave it for now.